package.json

```diff
@@ -1,31 +1,30 @@
 {
   "name": "@liftedinit/many-js",
-  "version": "0.1.0-alpha.7",
+  "version": "0.1.0-alpha.8",
```

Comment:
I'd prefer to bump the version of `many-js` separately so we generate tags and GitHub releases.

Reply:
I agree, also we can add to a document as part of the process of delivering a new release explaining the steps. I'll remove it from this pr.